### PR TITLE
fix: amend orcid works

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

File: packages/react-components/src/organisms/ProfileRecentWorks.tsx [hunk 1]
```diff
@@ -55,49 +59,65 @@ const typeMap: { [key in OrcidWork['type']]: string } = {
 };
 
 const containerStyles = css({
-  display: 'flex',
-  flexDirection: 'row',
-  flexWrap: 'wrap',
-  marginLeft: '-8px',
```
Comment (jeysal) [inline packages/react-components/src/organisms/ProfileRecentWorks.tsx:61]: You're lucky I never came across this :axe:

File: packages/react-components/src/organisms/ProfileRecentWorks.tsx [hunk 2]
```diff
@@ -55,49 +59,65 @@ const typeMap: { [key in OrcidWork['type']]: string } = {
...
-  marginRight: '-8px',
+  display: 'grid',
+  gridRowGap: `${12 / perRem}em`,
 });
 
-const elementStyle = css({
-  paddingLeft: '8px',
-  paddingRight: '8px',
+const elementStyle = css({});
```
Comment (jeysal) [inline packages/react-components/src/organisms/ProfileRecentWorks.tsx:66]: ..

**Review by jeysal — APPROVED** [review]
coverage increase <3 :D

File: packages/react-components/src/organisms/ProfileRecentWorks.tsx (hunk 1 above)
Comment (fampinheiro) [reply packages/react-components/src/organisms/ProfileRecentWorks.tsx:61]: If it works it ain't stupid.

File: packages/react-components/src/organisms/ProfileRecentWorks.tsx (hunk 2 above)
Comment (fampinheiro) [reply packages/react-components/src/organisms/ProfileRecentWorks.tsx:66]: Where does that come from :thinking:

**Pull request merged**